docs/EmbeddedAppIOS.md

```diff
@@ -146,6 +148,32 @@ Then add it as a subview of the `ReactView`.
 rootView.frame = self.bounds;
 ```
 
+> Or for Swift apps, on the ReactView.swift file add the following:
```

Comment:
I think instead of having it in a quote, it makes more sense to have anther heading for this.